shell/platform/tizen/flutter_tizen.cc

```diff
@@ -10,53 +10,37 @@
 #include "flutter/shell/platform/common/cpp/client_wrapper/include/flutter/plugin_registrar.h"
 #include "flutter/shell/platform/common/cpp/client_wrapper/include/flutter/standard_message_codec.h"
 #include "flutter/shell/platform/common/cpp/incoming_message_dispatcher.h"
+#include "flutter/shell/platform/tizen/flutter_tizen_engine.h"
 #include "flutter/shell/platform/tizen/public/flutter_platform_view.h"
 #include "flutter/shell/platform/tizen/public/flutter_tizen_texture_registrar.h"
-#include "flutter/shell/platform/tizen/tizen_embedder_engine.h"
 #include "flutter/shell/platform/tizen/tizen_log.h"
 
-// Opaque reference to a Tizen embedder engine.
-struct FlutterWindowControllerState {
-  std::unique_ptr<TizenEmbedderEngine> engine;
-};
-
-FlutterWindowControllerRef FlutterCreateWindow(
-    const FlutterEngineProperties& engine_properties) {
-  StartLogging();
-
-  auto state = std::make_unique<FlutterWindowControllerState>();
-  state->engine = std::make_unique<TizenEmbedderEngine>();
-
-  if (!state->engine->RunEngine(engine_properties)) {
-    FT_LOGE("Failed to run the Flutter engine.");
-    return nullptr;
-  }
+// Returns the engine corresponding to the given opaque API handle.
+static FlutterTizenEngine* EngineFromHandle(FlutterDesktopEngineRef ref) {
```

Comment:
I'm used to the expression below, what do you think?
```suggestion
// Returns the engine corresponding to the given opaque API handle.
static FlutterTizenEngine* ToImpl(FlutterDesktopEngineRef ref) {
```

Reply:
The name is from the windows implementation:

https://github.com/flutter-tizen/engine/blob/ad1fd5badb74632ecc88343101ff2f601574e5b4/shell/platform/windows/flutter_windows.cc#L30-L40

If `ToImpl` feels comfortable for you, I can change to it.

Reply:
Oh, I didn't know. I think the way I'm used to it is more intuitive, but no problem. 
Let's leave it as it is for the unification of terms.